### DISCUSSION TITLE
PathVisio file is lost

### DISCUSSION THREAD

**Discussion by JuliaBatUM** [wikipathways/wikipathways-help#60] [Q&A] [answered]
Hi!
I used an autosaved file for the last few weeks (PathVisioAutoSave.gpml), and now i cannot find it anymore on my computer. Also when I go on **open recent** in PathVisio, I can still see the file name but I cannot open it. I get this error message:
![WhatsApp Image 2023-08-18 at 17 38 46](https://github.com/wikipathways/wikipathways-help/assets/130973169/eeca7b01-6dd7-4ef5-beaf-bdc1a030cd43).
Is there a way to get back the file? I already navigated to the mentioned file name  C:\Users\jubo0\AppData\Local\Temp, there was a PathVisio file that could not be opened on PathVisio. 
Thank you very much for your help,
Julia

**Comment by khanspers** [comment, marked as answer]
Hi @JuliaBatUM. A couple of things to try:
- Try to find the PathVisio log file mentioned in the error message. According to the [PathVisio Troubleshooting FAQs](https://pathvisio.org/faq#troubleshooting), it should be in “AppData”- “Roaming” - “PathVisio”. If you find the error log, add a screenshot of it here.
- You mentioned there is a file in C:\Users\jubo0\AppData\Local\Temp that could not be opened in PathVisio, is it a gpml file? Or what is the file extension? 

Maybe others on the team have more experience with running PathVisio on Windows and may be able to help: @mkutmon @egonw @DeniseSl22

**Reply by JuliaBatUM** [reply]
Hi,
Thank you for your help.
I just spoke with @mkutmon and she told me that the file is gone as "Temp" is a temporary file.
Kind regards,
Julia

**Reply by mkutmon** [reply]
Yes, this is a Windows temporary folder that gets automatically cleaned up after a certain time. Make sure you always save your GPML files in a persistent location (maybe even a Dropbox or Gdrive folder so there is an additional backup) while you work locally on your pathways.

**Comment by mkutmon** [comment]
I will check if we can somehow notify users that autosave is just a temp save in PathVisio itself.
